docs/guides/references/changelog.mdx

```diff
@@ -2,6 +2,16 @@
 title: Changelog
 ---
 
+## 12.6.0
+
+_Released 02/14/2023_
+
+**Features:**
```

Comment:
I think this is missing the option to specify auto cancellation setting via CLI added in PR cypress-io/cypress#25237. This PR #5032 is for the relevant new section in docs.
The update to the CLI, module API, and error messages are accounted for, it is just missing here in the list of features.

Reply:
@MuazOthman This hasn't been updated with the cypress repo yet